custom_connectors/custom_auth/kizen.rb

```diff
@@ -56,6 +56,119 @@
       input
     end,
 
+    format_payload: lambda do |payload|
+      if payload.is_a?(Array)
+        payload.map do |array_value|
+          call('format_payload', array_value)
+        end
+      elsif payload.is_a?(Hash)
+        payload.map do |key, value|
+          key = call('inject_special_characters', key)
+          if value.is_a?(Array) || value.is_a?(Hash)
+            value = call('format_payload', value)
+          end
+          { key => value }
+        end.inject(:merge)
+      end
+    end,
```

Comment:
This has been replaced with the `format_input` method

Reply:
deleted format_payload method and changed to format_input